src/lint/rules/mod.rs

```diff
@@ -56,6 +56,7 @@ pub trait Rule {
                 range,
                 description: Self::description().to_string(),
             },
+            rule_ident: Self::name(),
             rule_type: Self::get_rule_type(),
         }
```

Comment:
Do we want the user to see the same identifier used for the lint_cfg.json file?

That seems like the most convenient way to correlate a warning message with its identifier, so a user can disable or use some in-code annotation to bypass a given rule.

I can update the name() functions for each rule to return this identifier.

Reply:
https://github.com/intel/dml-language-server/pull/125 you can merge this if you agree

Reply:
> Do we want the user to see the same identifier used for the lint_cfg.json file?
> 
> That seems like the most convenient way to correlate a warning message with its identifier, so a user can disable or use some in-code annotation to bypass a given rule.
> 
> I can update the name() functions for each rule to return this identifier.

Yes this was the idea, thanks for the fix.